docker-centos/Dockerfile

```diff
@@ -1,12 +1,20 @@
 FROM centos
 
-LABEL maintainer="Giuseppe Scrivano <gscrivan@redhat.com>"
+LABEL maintainer="Giuseppe Scrivano <gscrivan@redhat.com>" \
+      name="docker-centos" \
+      version="0.1" \
+      atomic.type="system" \
+      architecture="x86_64"
 
-RUN yum install -y docker container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin docker-lvm-plugin lvm2 iptables procps-ng xz inotify-tools && mkdir -p /usr/lib/modules && yum-config-manager --nogpgcheck --add-repo https://cbs.centos.org/repos/virt7-docker-el-candidate/x86_64/os/ && yum install -y --nogpgcheck docker-rhel-push-plugin && yum clean all
+RUN yum install --setopt=tsflags=nodocs -y docker container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin docker-lvm-plugin lvm2 iptables procps-ng xz inotify-tools && \
+mkdir -p /usr/lib/modules && \
+yum-config-manager --nogpgcheck --add-repo https://cbs.centos.org/repos/virt7-docker-el-candidate/x86_64/os/ && \
+yum install --setopt=tsflags=nodocs -y --nogpgcheck docker-rhel-push-plugin && \
+yum clean all
 
 ADD init.sh /usr/bin
 
 # system container
-COPY service.template tmpfiles.template config.json.template /exports/
+COPY daemon.json service.template tmpfiles.template config.json.template /exports/
```

Comment:
Newer versions of atomic can copy arbitrary files to the host.  It is enough to copy the `daemon.json` file under `/exports/hostfs/`. e.g. `/exports/hostfs/etc/foo` is copied to the host `/etc/foo`

We can make the file name depend from the container name with:
```
  "renameFiles" : {
        "/etc/daemon.json" : "/etc/sysconfig/daemon-$NAME.json"
    },
```

It depends if we want to allow multiple versions of the Docker container to be installed at the same time, or have different configurations.  As we could just use a single file, and the container itself could verify if it exists on startup and create the file if it doesn't exist.